modules/Kconfig.mcuboot

```diff
@@ -128,12 +130,21 @@ config MCUBOOT_EXTRA_IMGTOOL_ARGS
 	  string) you can use this option to pass extra options to
 	  imgtool. For example, you could set this to "--version 1.2".
 
+config MCUBOOT_SUPPRESS_WARNING_IMAGE_NOT_SIGNED
+	bool "Suppress warning about image not being signed"
+	depends on MCUBOOT_SIGNATURE_KEY_FILE = "" && !MCUBOOT_GENERATE_UNSIGNED_IMAGE
+	help
+	  If enabled, suppress the warning about the image not being signed/prepared
+	  when CONFIG_MCUBOOT_SIGNATURE_KEY_FILE is an empty string
+
```

Comment:
we don't need this, it outputs a warning indicating the status which can be ignored if it's expected

Reply:
removed that, now only the change of the help text is lest